docs/04 - Model Tab.md

```diff
@@ -46,6 +46,7 @@ Examples:
 * **cfg-cache**: Creates a second cache to hold the CFG negative prompts. You need to set this if and only if you intend to use CFG in the "Parameters" > "Generation" tab. Checking this parameter doubles the cache VRAM usage.
 * **no_flash_attn**: Disables flash attention. Otherwise, it is automatically used as long as the library is installed.
 * **cache_8bit**: Create a 8-bit precision cache instead of a 16-bit one. This saves VRAM but increases perplexity (I don't know by how much).
+* **cache_q4**:Creates a Q4 cache using grouped quantization instead of the standard 16-bit one. Recommended over 8-bit cache, as it's much smarter with the size trimming and results in better performance at lower usage. |
```

Comment:
Same as the comment above, I don't have data to base this claim. Also missing a space after `**cache_q4**:`